hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/DeltaSync.java

```diff
@@ -299,11 +302,17 @@ private void refreshTimeline() throws IOException {
       schemaProvider = dataAndCheckpoint.getSchemaProvider();
     }
 
-    if ((!avroRDDOptional.isPresent()) || (avroRDDOptional.get().isEmpty())) {
-      log.info("No new data, nothing to commit.. ");
+    if (Objects.equals(checkpointStr, resumeCheckpointStr.orElse(null))) {
+      log.info("No progress in source consumption. Old checkeckpoint=(" + resumeCheckpointStr
```

Comment:
can we keep "No new data, source checkpoint has not changed. Nothing to commit." as the message. typo in "Old checkpoint".

Reply:
Done